cs-base-station-interface/src/main/java/basestation/vision/OverheadVisionSystem.java

```diff
@@ -31,7 +31,9 @@ public OverheadVisionSystem() {
     private void processBlobs(Blob[] data) {
         HashSet<VisionObject> newSet = new HashSet<>();
         for (Blob b : data) {
-            VisionCoordinate vc = new VisionCoordinate(b.x, b.y, ((((b.orientation % (2 * Math.PI)) * 180 / (Math.PI))) + 180) % 360);
+            VisionCoordinate vc = new VisionCoordinate(b.x, b.y, Math
```

Comment:
reject this change when merging develop- I tested the conversion to follow the desired coordinate system.